rpm-readme.md

```diff
@@ -56,19 +56,39 @@ Done.
 
 ```
 
-#### Installing rpm package
+> You may want to check RPM package content with `sudo rpm -ql foglamp-gui-x.y.z-1.x86_64.rpm`
```

Comment:
- Put `-p` option as well to check the content  of uninstalled RPM package. 
e.g. `$ sudo rpm -qplv foglamp-gui-x.y.z-1.x86_64.rpm`

- To check installed package content then use `-q` option only with no package extension (.rpm)
e.g. `sudo rpm -ql foglamp-gui-x.y.z-1.x86_64`

options
`-q` - to specify it as a query command
`-l` - to list the files in the package
`-p` -  to query the uninstalled package file.
`-v` - (verbose) to provides additional information (permissions, owner, etc.)

Reply:
Fixed!